.github/workflows/deploy_mongo_to_pg.yml

```diff
@@ -0,0 +1,51 @@
+name: Deploy
+
+run-name: Deploy ${{ inputs.gitRef || github.event.release.tag_name  }} to ${{ inputs.environment || 'integration' }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      gitRef:
+        description: 'Commit, tag or branch name to deploy'
+        required: true
+        type: string
+      environment:
+        description: 'Environment to deploy to'
+        required: true
+        type: choice
+        options:
+        - integration
+        default: 'integration'
+      ecrRepositoryName:
+        description: 'ECR repo name to push image to'
+        required: true
+        type: choice
+        options:
+          - publisher-on-postgres-branch
+        default: 'publisher-on-postgres-branch'
```

Comment:
```suggestion
```
As there's only one ECR Repo that you intend to target, I'd consider removing this option.